config.yml

```diff
@@ -4,10 +4,10 @@ feeds:
     format: "📝 New blog post Released!\n{link}\n\n{content}"
 
     media:
-      - mastodon-eu-freiburg
-      - matrix-eu-announce
-      - bluesky-galaxyproject
-      - linkedin-galaxyproject
+      - - mastodon-eu-freiburg
+        - matrix-eu-announce
+        - linkedin-galaxyproject
+      - - bluesky-galaxyproject
```

Comment:
two dashes?

Reply:
this will be parsed into this:
```json
[
  [
    "mastodon-eu-freiburg",
    "matrix-eu-announce",
    "linkedin-galaxyproject"
  ],
  [
    "bluesky-galaxyproject"
  ]
]
```

Reply:
If this is confusing we can use this format:
```yaml
      - mastodon-eu-freiburg:
        matrix-eu-announce:
        linkedin-galaxyproject:
      - bluesky-galaxyproject:
```

Reply:
or this format:
```yaml
    1:
      - mastodon-eu-freiburg
      - matrix-eu-announce
      - linkedin-galaxyproject
    2:
      - bluesky-galaxyproject
```

Reply:
can we have something like:

```
- group1:
  - one
  - two
  - three
- group2
  - four 
```

And then a fat comment above that describes why ...